lib/instrumentation/run-context/AbstractRunContextManager.js

```diff
@@ -29,10 +29,10 @@ const ADD_LISTENER_METHODS = [
 // The implementation is adapted from
 // https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-context-async-hooks/src/AbstractAsyncHooksContextManager.ts)
 class AbstractRunContextManager {
-  constructor(log, runContextClass = RunContext) {
+  constructor(log, RunContextClass = RunContext) {
     this._log = log;
     this._kListeners = Symbol('ElasticListeners');
-    this._root = new runContextClass(); // eslint-disable-line new-cap
+    this._root = new RunContextClass();
```

Comment:
nit: Not sure I love this change. :)  Obviously it is minor, but `runContextClass` is a *variable* holding a class.

Reply:
no problem, I'll revert it :)

Reply:
done